README.md

```diff
@@ -1,22 +1,27 @@
 
  In this article, we are concentrating on unit testing, a type of functional testing.
  
  ## What is Unit testing? 
-Unit testing is a form of software testing that focuses on validating individual units or components of a software application and confirming that each unit functions as intended. It is considered the first phase of functional testing in the software development life cycle. It ensures that it has no impact on any other parts of your code than the component or unit under test by isolating them from the rest of your code. As a result, it isolates a single functional component, guarantees correctness, and confirms that there are no errors, enabling speedy bug fixes before merging them into the main code. It is a type of "white box" testing. It is frequently accomplished by writing test cases that test specific functions, methods, and classes. Test cases can be written in plain English, even though they are usually written in pseudocode (implemented in a language like Java or JavaScript).
+Unit testing is a form of software testing that focuses on validating individual units or components of a software application and confirming that each unit functions as intended ([ref?]()). It is considered the first phase of functional testing in the software development life cycle ([ref?]()). It ensures that it has no impact on any other parts of your code than the component or unit under test by isolating them from the rest of your code ([ref?]()). As a result, it isolates a single functional component, guarantees correctness, and confirms that there are no errors _\[from: https://www.lambdatest.com/learning-hub/unit-testing\]_, enabling speedy bug fixes before merging them into the main code ([ref?]()). It is a type of "white box" testing. It is frequently accomplished by writing test cases that test specific functions, methods, and classes ([ref?]()). Test cases can be written in plain English, even though they are usually written in pseudocode (implemented in a language like Java or JavaScript) ([ref?]()).
 
 The procedure for conducting unit tests consists of four steps:
-- Creating test cases: Write several test cases for each component of a web application.
+1. Creating test cases: Write several test cases for each component of a web application.
```

Comment:
Use enumerated/numeric lists where appropriate (like when you say there will be 'four steps').